### ISSUE TITLE
linkname not honored when releasefilename not default

### ISSUE THREAD

**Issue by dmolik** [rjbrown57/binman#61]
when the release filename doesn't match the repo name in a release archive, a custom linkame is not honored.

this works:

```
- repo: grafana/loki
  url: https://github.com/grafana/loki/releases/download/{{ .version }}/loki-{{ .os }}-{{ .arch }}.zip
  releasefilename: loki-linux-amd64
  linkname: loki
```

this does not:

```
- repo: grafana/loki
  url: https://github.com/grafana/loki/releases/download/{{ .version }}/loki-{{ .os }}-{{ .arch }}.zip
  linkname: loki
```

**Comment by rjbrown57** [comment]
Thanks @dmolik i'll get this fixed up!

**Comment by rjbrown57** [comment]
This is fixed in v0.5.2
